src/app/client/src/app/modules/workspace/components/create-editors/collection-editor/collection-editor.component.ts

```diff
@@ -226,44 +199,47 @@ export class CollectionEditorComponent implements OnInit, AfterViewInit {
       });
     }
     window.config.editorConfig.publishMode = false;
-    window.config.editorConfig.isFalgReviewer = false;
-    if (this.state === 'UpForReviewContent' &&
+    window.config.editorConfig.isFlagReviewer = false;
+
+    const enum state {
```

Comment:
Define enum in a separate file and use where required.

Reply:
done